### PR TITLE
Update Arduino boards config.

### PR DESCRIPTION
**Pull request by iabdalkader**
This PR includes misc small fixes and updates to Portenta-H7 and Nano RP2040 board config files.

Depends on:
~- [ ] https://github.com/micropython/micropython-lib/pull/541~
- [x] https://github.com/micropython/micropython-lib/pull/554
- [x] https://github.com/micropython/micropython-lib/pull/555

### CODE REVIEW

**Comment by iabdalkader** [comment]
Rebased.

File: ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/manifest.py [hunk 1]
```diff
@@ -3,9 +3,12 @@
 # Networking
 require("webrepl")
 require("urequests")
+require("ntptime")
+require("kpn-senml")
```
Comment (dpgeorge) [inline ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/manifest.py:7]: I suggest removing this kpn-senml line for now, then this PR can be unblocked and merged.

**Comment by ubidefeo** [comment]
@iabdalkader 
can we also update the `dfu-util` burn command?
It now fails on MacOS and Windows.

from
`dfu-util -w -d 2341:035b -a 0 -d 0x8040000 -D firmware.dfu`
to
`dfu-util -a 0 -d 0x2341:0x035b -D firmware.dfu`

the `0x` prefix on the VID and PID doesn't seem to matter, but I find it more visually consistent for a 2 Byte HEX.

the `-d` flag will only recognise a pair, but `0x80400000` might to be an offset I've seen in the past, some remnant from another recipe for the H7's partitioned flash with a bootloader.
It seem that it won't fail on Linux (it did work on an Arch Linux machine)

File: ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/manifest.py (hunk 1 above)
Comment (iabdalkader) [reply ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/manifest.py]: I removed `kpn-senml` and squashed the commits. I also added `logging` to manifests, it's limited but this way when it gets updated in `micropython-lib` it's going to be enabled.

**Comment by dpgeorge** [comment]
Merged in 8ddb4567ff94c58e56a23c652a7323575f0079d1 and e35bcb0f0b940ffb73396cd6c023ac29b16e5b9b

**Pull request closed without merging**